### ISSUE TITLE
Maybe this validation is not needed

### ISSUE THREAD

**Issue by juanmbellini** [juanmbellini/PowerUp#57]
https://github.com/juanmbellini/PowerUp/blob/9d60b36295dcf457ea8c11c752f58afca4eb8078/paw-webapp/model/src/main/java/ar/edu/itba/paw/webapp/model/Comment.java#L98

When creating a reply, a parent ```Comment``` is set, and its validated to check no cycle is created by checking that getting a comment's parent, the given comment (```this```) is not reached.
The thing is that, as the comment is not created yet (this is done in the constructor), it can not be set as a parent comment on any other comment, so the ```Comment#validParent``` will always return ```true```.

**Comment by lipusal** [comment]
[El blame dice que vos implementaste este método](https://github.com/juanmbellini/PowerUp/blame/9d60b36295dcf457ea8c11c752f58afca4eb8078/paw-webapp/model/src/main/java/ar/edu/itba/paw/webapp/model/Comment.java#L283), así que vos arreglalo cuando te liberes. Me parece válido arreglar esto, nunca van a ser iguales porque todavía no tiene ID y el `equals` es por ID.

**Comment by juanmbellini** [comment]
Claro, es verdad que además el equals es por id, pero además, es imposible que ocurra (se puede ver por inducción (? ajajaja)
